Makefile

```diff
@@ -45,7 +47,7 @@ GIT_SUBMODULE_UPDATE := git submodule update --init --recursive
 else # "variables.mk" was included. Business as usual until the end of this file.
 
 # default target, because it's the first one that doesn't start with '.'
-all: | wakunode wakusim wakunode2 wakusim2
+all: | wakunode wakusim wakuexample wakunode2 wakusim2
```

Comment:
Maybe this should be explicit about being version1? WDYT?

Also would be nice to add to README how to run examples.

Reply:
Lets do some further make file cleanup with explicit versions in another PR.

I'll add a small readme instead of the inline explanation.